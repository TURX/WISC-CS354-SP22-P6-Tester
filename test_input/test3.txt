0 500 32