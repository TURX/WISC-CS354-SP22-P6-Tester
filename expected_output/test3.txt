Reading character starting at index 0 to 500 with an interval of 32

Reading character at index 0   : data = A : hit count = 0   : miss count = 1   : read data count = 1
Reading character at index 32  : data =
 : hit count = 0   : miss count = 2   : read data count = 2
Reading character at index 64  : data = U : hit count = 0   : miss count = 3   : read data count = 3
Reading character at index 96  : data =   : hit count = 0   : miss count = 4   : read data count = 4
Reading character at index 128 : data =   : hit count = 0   : miss count = 5   : read data count = 5
Reading character at index 160 : data = U : hit count = 0   : miss count = 6   : read data count = 6
Reading character at index 192 : data =   : hit count = 0   : miss count = 7   : read data count = 7
Reading character at index 224 : data =   : hit count = 0   : miss count = 8   : read data count = 8
Reading character at index 256 : data = H : hit count = 0   : miss count = 9   : read data count = 9
Reading character at index 288 : data = g : hit count = 0   : miss count = 10  : read data count = 10
Reading character at index 320 : data =   : hit count = 0   : miss count = 11  : read data count = 11
Reading character at index 352 : data = n : hit count = 0   : miss count = 12  : read data count = 12
Reading character at index 384 : data =   : hit count = 0   : miss count = 13  : read data count = 13
Reading character at index 416 : data = r : hit count = 0   : miss count = 14  : read data count = 14
Reading character at index 448 : data =   : hit count = 0   : miss count = 15  : read data count = 15
Reading character at index 480 : data =   : hit count = 0   : miss count = 16  : read data count = 16
